tests/src/test/java/alluxio/shell/command/LsCommandTest.java

```diff
@@ -249,4 +249,22 @@ public void lsr() throws Exception {
         files[3].isFolder());
     Assert.assertEquals(expected, mOutput.toString());
   }
+
+  /**
+   * Tests ls command with a file where the file name includes a specifier character.
+   */
+  @Test
+  @LocalAlluxioClusterResource.Config(
+      confParams = {PropertyKey.Name.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, "false",
+          PropertyKey.Name.SECURITY_AUTHENTICATION_TYPE, "NOSASL"})
+  public void lsWithFormatSpecifierCharacter() throws IOException, AlluxioException {
+    String fileName = "/localhost%2C61764%2C1476207067267..meta.1476207073442.meta";
+    FileSystemTestUtils.createByteFile(mFileSystem, fileName, WriteType.MUST_CACHE, 10);
+    URIStatus file = mFileSystem.getStatus(new AlluxioURI(fileName));
+    mFsShell.run("ls", "/");
+    String expected =
+        getLsNoAclResultStr("/localhost%2C61764%2C1476207067267..meta.1476207073442.meta",
```

Comment:
lets replace the string with `fileName` to make it easier to tell that they are the same